examples/tv-app/android/java/OnOffManager.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <jni.h>
+
+class OnOffManager
```

Comment:
please add some documentation of what this class does and why it exists.

From the naming, just having "It manages On Off" does not seem obvious.

Reply:
fixed

Reply:
Sorry, I am not a native English speaker,  I tried to copy and use short sentences but I am not sure if it is appropriate.

- InitializeWithObjects is a common name for all java bridges in this project to get jni object/class/methods
- NewManager is a common name for all cluster java bridges in this project to  install for one endpoint
so I just want to keep them same

And for InitializeWithObjects, I have changed it to private now.